recipes/sagemaker-code-editor/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "sagemaker-code-editor" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz  # [linux64]
+  sha256: c3cce647409ab64980aa052188b710911b0e83af524b23e4e419125733b4dfe4  # [linux64]
+  folder: sagemaker-code-editor
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - sysroot_linux-64 2.17  # [build_platform == 'linux-64']
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+    - nodejs 18.*
+    - openssl
+    - jq
+    - glib         # [linux]
+    - libsecret    # [linux]
+    - libxkbfile   # [linux]
+    - xorg-libx11  # [linux]
+    # node-gyp is broken with python 3.12 - https://github.com/nodejs/node-gyp/issues/2869, so pulling in distutils
+    - setuptools
+  host:
+    - nodejs 18.*
+    - openssl
+    - yarn <2
+    - ripgrep
+    - glib         # [linux]
+    - libsecret    # [linux]
+    - libxkbfile   # [linux]
+    - xorg-libx11  # [linux]
+    # node-gyp is broken with python 3.12 - https://github.com/nodejs/node-gyp/issues/2869, so pulling in distutils
+    - setuptools
+  run:
+    - nodejs 18.*
```

Comment:
I think that it is not necessary to specify nodejs as a runtime requirement because it is already a host dependency, and nodejs is specified as a `run_export`.

```
  run_exports:
    - {{ pin_subpackage('nodejs', max_pin='x') }}
```